packages/neon/neon/lib/src/router.dart

```diff
@@ -105,6 +116,14 @@ class HomeRoute extends GoRouteData {
       path: 'flow/:serverURL',
       name: 'loginFlow',
     ),
+    TypedGoRoute<LoginQrcodeRoute>(
+      path: 'qrcode',
+      name: 'loginQrcode',
+    ),
+    TypedGoRoute<LoginQrcodeIntermediateRoute>(
+      path: 'qrcode/intermediate/:serverURL/:loginName/:password',
```

Comment:
wouldn't it make sense to just call this the same as the qr code encodes?

THis way we wouldn't need the global redirect (wich AFAICT gets executed on every navigation)

Reply:
As mentioned above that doesn't work because the URI is stripped. It would just be one big parameter which really sucks. Not a problem of our architecture but of the structure of the QR code login URIs.

Reply:
can we think about #144 nevertheless?
I don't like the `/` between the parameters.

We can also do all at once with the mentioned issue.